woodwork/serialize.py

```diff
@@ -48,7 +48,8 @@ def typing_info_to_dict(dataframe):
          'physical_type': {
              'type': str(dataframe[col_name].dtype),
              # Store categorical values so they can be recreated if they are modified during serialization
-             'cat_values': dataframe[col_name].dtype.categories.to_list() if str(dataframe[col_name].dtype) == 'category' else None
+             'cat_values': dataframe[col_name].dtype.categories.to_list() if str(dataframe[col_name].dtype) == 'category' else None,
+             'cat_dtype': str(dataframe[col_name].dtype.categories.dtype) if str(dataframe[col_name].dtype) == 'category' else None
```

Comment:
could we omit these keys if the column is not categorical?

Reply:
Yes - the keys aren't needed otherwise. I'll try to clean that up.